src/angular-cli-ghpages

```diff
@@ -44,12 +46,20 @@ commander
   )
   .option(
     '-S, --no-silent',
-    'Logging is in silent mode by default. The option enables extended console logging. Keep this untouched if the repository URL or other information passed to git commands is sensitive!'
+    'Deprecated! This paremeter is no longer needed. It will be ignored.'
   )
   .option(
     '-T, --no-dotfiles',
     'Includes dotfiles by default. Otherwise files starting with `.` are ignored.'
   )
+  .option(
+    '--no-notfound',
+    'By default a 404.html file is created, because this is the only known workaround to avoid 404 error messages on GitHub pages. For Cloudflare Pages we highly recommend to disable the 404.html file by setting this switch to true!',
```

Comment:
```suggestion
    'By default a 404.html file is created, because this is the only known workaround to avoid 404 error messages on GitHub Pages. For Cloudflare Pages we highly recommend to disable the 404.html file by setting this switch to true!',
```